tdwii_plus_examples/cli/upscreatescu.py

```diff
@@ -120,43 +120,45 @@ def main():
     # Verification requested
     if args.echo:
         print("Verification (C-ECHO) request")
-        result = scu.verify()
-        if result.status_category == "Success":
+        num_created_instances = scu.verify()
```

Comment:
this looks like it should not have changed, because it was a result, not a count.